### ISSUE TITLE
release schedule

### ISSUE THREAD

**Issue by vasilich6107** [salim-lachdhaf/searchable_dropdown#210]
There is a know issue of this lib that many fixes are not available to users due to long lasting release cycle.
This is why I was forced to maintain my clone https://pub.dev/packages/dropdown_search2

Currently `beta` is non suitable for production and `stable` version lacks of many features.

Here is a lib where each merge into dev tests are running and produces release.
https://github.com/comigor/artemis/

Could you organise something similar to release features and bugfixes more frequently?

**Comment by salim-lachdhaf** [comment]
1- why you are seeing that beta version is not suitable for production ?
2- which feature the stable version is lacking ?
3- i can added actions on git to test the PR , i already did that for other libs ..., but accepting a PR did not mean release a new version !

**Comment by vasilich6107** [comment]
1. It is `beta` and it is has some chance of having bugs. I'm not pretending to rely on prefixes. Even stable can contain bugs. I did my assumption on that fact that I've found some of them while trying to update `reactive_serchable_dropdown`
2. I double checked and it seems that `0.6.3` contains all fixes that we did. Nevertheless there was times when users have to wait for long time between merging the PR and release. For example https://github.com/salim-lachdhaf/searchable_dropdown/pull/139 was merged on 29 Mar and released in v0.6.1 in May 2, 2021
3. I understand that there could be minor fixes in readme that does not deserve release. To my mind it is better to have a strategy 1 PR - 1 release to rollout fixes as quickly as possible rather than making users to wait.

To summarise - all of this are my suggestions that highlights my pains during usage of the library and aim to improve user experience

**Comment by vasilich6107** [comment]
Also while having pipeline we could start adding some tests to make releases more reliable.

**Comment by salim-lachdhaf** [comment]
in my pipline, i have planned to add git actions to handle testing PR and publishing versions ...

but i want to mention that the reason behind publishing a beta version and asking users for possible updates ... is that i know that it will be some minor updates or bugs on that version. because i didn't tested it very well ... because sadly my time is extremly limited , and it's the reason for all delays in PR testing and accepting and releasing versions and bug fixing ...
